salt/modules/cmdmod.py

```diff
@@ -2144,11 +2144,13 @@ def _cleanup_tempfile(path):
                     'stderr': '',
                     'cache_error': True}
         shutil.copyfile(fn_, path)
+
     if not salt.utils.is_windows():
         os.chmod(path, 320)
         os.chown(path, __salt__['file.user_to_uid'](runas), -1)
-
-    path = _cmd_quote(path)
+        path = _cmd_quote(path)
+    else:
+        path = '"{0}"'.format(path)
```

Comment:
This fix is wrong. The issue as already been resolved. See the below, which is at the top of cmdmod.py
```
if salt.utils.is_windows():
    from salt.utils.win_runas import runas as win_runas
    from salt.utils.win_functions import escape_argument as _cmd_quote
    HAS_WIN_RUNAS = True
else:
    from salt.ext.six.moves import shlex_quote as _cmd_quote
    HAS_WIN_RUNAS = False
```